### ISSUE TITLE
Feature request: themeVariables as object with values instead of array

### ISSUE THREAD

**Issue by comerc** [mzohaibqc/antd-theme-generator#17]
I want to pass themeVariables as object with values instead of array in options of generateTheme. It is allow to me use CSS variables from JS (as CSS-IN-JS,  for example) .

**Comment by mzohaibqc** [comment]
Can you add sample code for better understanding

**Comment by comerc** [comment]
Instead of this code:
```javascript
const options = {
  stylesDir: path.join(__dirname, './src/styles'),
  antDir: path.join(__dirname, './node_modules/antd'),
  varFile: path.join(__dirname, './src/styles/vars.less'),
  mainLessFile: path.join(__dirname, './src/styles/main.less'),
  themeVariables: [
    '@primary-color',
    '@secondary-color',
    '@text-color',
    '@text-color-secondary',
    '@heading-color',
    '@layout-body-background',
    '@btn-primary-bg',
    '@layout-header-background',
  ],
  indexFileName: 'index.html',
  outputFilePath: path.join(__dirname, './public/color.less'),
}
```
I want to use this code:
```javascript
const options = {
  stylesDir: path.join(__dirname, './src/styles'),
  antDir: path.join(__dirname, './node_modules/antd'),
  // varFile: path.join(__dirname, './src/styles/vars.less'),
  // mainLessFile: path.join(__dirname, './src/styles/main.less'),
  themeVariables: {
    '@primary-color': '#ddddd0',
    '@text-color': '#000000',
    '@text-color-secondary': '#eb2f96',
    '@heading-color': '#ccccdd',
    '@secondary-color': '#0000ff',
    '@layout-header-background': '#9e8989',
    '@btn-primary-bg': '#ccbb00',
    '@bg-color': '#ddddde',
  },
  indexFileName: 'index.html',
  outputFilePath: path.join(__dirname, './public/color.less'),
}
```
(themeVariables as object w/o varFile and mainLessFile)

Thanks!

**Comment by comerc** [comment]
(FYI) I implemented your example with CRA 2.0: https://github.com/comerc/minsk4/tree/antd-theme-generator

**Comment by mzohaibqc** [comment]
@comerc I am too busy these days, will try to add this soon
